src/data/graphql/queries.graphql

```diff
@@ -242,6 +242,9 @@ query ColonyAction($transactionHash: String!, $colonyAddress: String!) {
     domainName
     domainPurpose
     domainColor
+    motionId
+    motionDomainId
+    rootHash
```

Comment:
Why added them explicitly here?

You already have to `MotionCreated` event in the `events` object. Why not take your values from there ?

This is just pointless duplication!!!

Have a look at @alicjakujawa 's PR on how to do this: https://github.com/JoinColony/colonyDapp/blob/b04d8e6df8f93bf4ce07d621cb3c68b4f1b15c36/src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx#L65-L68